spec/classes/solr5_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe 'solr5', :type => 'class' do
+
+  # You need this, because otherwise puppetlabs-concat fails
+  let :facts do
+    {
+       :concat_basedir => 'blubber',
+     }
+  end
+
+  it{
+      should contain_class('solr5')
```

Comment:
@bjoernhaeuser In general, you are right, but the first essential part is that the test starts a compilation of the whole thing.